nestkernel/node.h

```diff
@@ -477,6 +477,77 @@ class Node
   virtual void handle( DoubleDataEvent& e );
 
   /**
```

Comment:
I think it would be very useful if you grouped all the getter-functions related to structural plasticity in a doxygen group and added a group-level explanation of their purpose and usage. You can do this with `@defgroup` and `@ingroup` Doxygen tags, see, e.g., `librandom/randomgen.h`. It would make it much easier for other developers to understand and use this new NEST component.